lib/WWW/Mechanize.pm

```diff
@@ -3111,6 +3111,67 @@ sub _die {
 
 __END__
 
+=head1 BEST PRACTICES
+
+The default settings which Mechanize has can get you up and running quickly,
```

Comment:
`s/which/that/` or maybe it would be best to just drop "which Mechanize has" as it doesn't add much.

Reply:
Good catch!  I dropped that part and force-pushed.